src/upload/sagas.js

```diff
@@ -167,7 +167,7 @@ export function* handlePollJobs({ payload: { programKey, jobId, bannerId } }) {
             id: programKey + Date.now(),
             bannerType: 'success',
             message: `Your ${jobNameInfo.jobName} job has has successfully completed.`,
-            linkMessage: 'Click here for result',
+            linkMessage: 'Click here for result.',
```

Comment:
I added a period here so that the message is more consistent with other messages in the alert.